.gitignore

```diff
@@ -4,6 +4,9 @@
 # Gradle work directory
 .gradle
 
+# Gradle cache
+.gradletasknamecache
```

Comment:
interesting, I never had this file. What creates this ? (hint: not Gradle)

Reply:
I think it's an intellij file. But I haven't checked.